.github/workflows/ci.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       - name: Tests in flink-kubernetes-operator
         run: |
           cd flink-kubernetes-operator
-          mvn integration-test -Dit.skip=false
+          mvn verify -Dit.skip=false
           cd ..
       - name: Tests in flink-kubernetes-webhook
```

Comment:
Could we also use the `mvn verify` in "Tests in flink-kubernetes-webhook" even though we do not have ITCase now?

Reply:
Yes, it is a good suggestion, I believe it is fine to use `verify` on webhook as well. I have updated.